### PR TITLE
Feature/mhip insights screen

### PR DESCRIPTION
**Pull request by aikewoody**
# Description

_Description of the feature or fix with link to the issue if existing_

## On what platform have you tested the change?

- [ ] Android - Emulator
- [ ] Android - Physical device
- [x] iOS - Emulator
- [ ] iOS - Physical device

## Screenshots

_Please attach screenshots showing the change if relevant_

## Checklist

- [ ] I have updated mockServer
- [ ] I've added analytics as relevant
- [ ] I have run `npm test`
- [ ] I have run `npm run test:i18n`

## Out of scope and potential follow-up

_Is there any related changes that you plan to do in a follow-up PR or voluntarily excluded from the scope?_

### CODE REVIEW

File: src/features/mental-health-playback/screens/MHPIntroductionScreen.tsx [hunk 1]
```diff
@@ -26,7 +26,7 @@ export default function MHPIntroductionScreen() {
       style={styling.backgroundWhite}>
       <Introduction />
       <View style={styling.padding}>
-        <Card style={styling.marginBottom}>
+        <Card backgroundColor="#F5F9FC" style={styling.marginBottom}>
```
Comment (selvach) [inline src/features/mental-health-playback/screens/MHPIntroductionScreen.tsx:29]: Minor tidy up: move this to a color const file.

**Review by selvach — APPROVED** [review]
Wow. This looks amazing!!!

File: src/features/mental-health-playback/screens/MHPIntroductionScreen.tsx (hunk 1 above)
Comment (aikewoody) [reply src/features/mental-health-playback/screens/MHPIntroductionScreen.tsx:29]: Agreed, so the thing is I don't want to spend much time on this before we had the meeting about what to do with the theme functionality

File: src/core/state/mental-health-playback/slice.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+import { IMentalHealthPlayback, IInsightsDict } from '@covid/core/state/mental-health-playback/types';
+import { IInsight } from '@covid/types/mental-health-playback';
+
+// @todo
+const insights: IInsight[] = [
+  {
+    activity_name: 'sleeping_well_less',
+    answers: [
+      {
+        title: 'No change',
```
Comment (selvach) [inline src/core/state/mental-health-playback/slice.ts:13]: The real endpoint will return t

itle: "NO_CHANGE" | "MORE" | "LESS

**Pull request merged**